change/@internal-react-components-7d75bb36-4e69-4ff0-a7c2-19955184c1ba.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Show file upload errors above sendbox in Composites containing Chat features",
```

Comment:
Thanks for updating this, for this change file however it relates to the react-components packlet - so try to scope the comment to the packlet the changelog refers to (i.e. we do not have Composites inside the react-components package)